packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -156,7 +160,9 @@ export class TdsDropdown {
       this.reset();
     }
     this.handleChange();
-    return this.selection;
+    /* This returns an array of object with a value and label pair. This is ONLY to not break the API. Should be removed for 2.0. */
+    // TODO - Clean up and just return this.value for 2.0
```

Comment:
Should we have this in a ticket also? Maybe good rule to only introduce a TODO together with a Jira ticket? Or use some kind of linting for it, https://github.com/maxprilutskiy/eslint-plugin-no-expired-todo-comments, this one checks for expiry date of TODO?

Reply:
I can add it as a ticket also maybe? Feel like it could be good to have it here also to not forget it, feels like things gets lost in the backlog